portal/patient/scripts/app/onsitedocuments.js

```diff
@@ -436,6 +444,12 @@ var page = {
                     pageAudit.onsitePortalActivity.set('status', 'waiting');
                     page.onsiteDocument.set('denialReason', 'In Review');
                     page.updateModel(true);
+                    if (autoRender + auditRender > 0) {
+                        auditRender = 0;
+                        autoRender = 0;
+                        location.assign("https://opensourcedemr.us/portal/patient/onsitedocuments?pid=" + encodeURIComponent(cpid));
```

Comment:
odd seeing hard-coded server name here `https://opensourcedemr.us`

Reply:
No clue how that got there. Glad you caught it though but I'll probably caught it on my docker